masl.src.html

```diff
@@ -34,37 +33,38 @@ <h2>Introduction</h2>
       <h2>How to use a MASL document</h2>
       <p>
         The recommended structure for DASL metadata is to insert a dCBOR42 document "between" each
-        CID and its resource(s), essentially using dCBOR42 as headers wrapping a CID-addressed resource.
+        CID and its resource(s), essentially using dCBOR42 as headers wrapping one or more CID-addressed resource(s).
```

Comment:
```suggestion
        CID and its resource(s), essentially using dCBOR42 as headers wrapping one or more CID-addressed resources.
```

One or more is plural no?